Library/Homebrew/cask/test/cask/cli/reinstall_test.rb

```diff
@@ -0,0 +1,10 @@
+require "test_helper"
+
+describe Hbc::CLI::Reinstall do
+
+  it "allows reinstalling a Cask" do
+    Hbc::CLI::Install.run("local-transmission")
+    Hbc::CLI::Reinstall.run("local-transmission")
+  end
```

Comment:
This test should check if the files in `local-transmission` are actually installed.

They have to be installed after running `Install`, and they also have to be installed after running `Reinstall`.

There should be an equivalent test for the case where the cask is *not* already installed when running `Reinstall`.